src/cascade/saver/db/save_model_results_to_db.py

```diff
@@ -64,8 +64,23 @@ def _normalize_draws_df(draws_df, execution_context):
     )
 
     age_groups = get_age_groups(execution_context)
+
+    # TODO: Age group 164 (Birth) is a special case which is not included in the standard
+    # age group set and which the model may optionally produce for prevalence (and no other
+    # measure). I'm hard coding it in here but it really should be plumbed through better.
+    age_groups = age_groups.append([{
+        "age_group_id": 164,
+        "age_group_years_start": 0,
+        "age_group_years_end": 0,
+    }])
+
+    # TODO: This assumes there will never be two different age groups with the same
+    # upper range. Be wary
```

Comment:
I'm OK with that restriction if these are draws b/c we make the draws. They won't have age ranges that are summaries, combinations of ages together.

Reply:
I am conscious of it because I just add the birth age group which shares it's starting age with the neo-natal group and that broke some assumptions.